docs/examples.md

```diff
@@ -74,10 +81,21 @@ def plot(name: str) -> None:
 for name in [
     "split",
     "north-pole",
-    "both-poles",
     "complex-split",
     "multi-split",
     "overlap",
 ]:
     plot(name)
 ```
+
+## Winding order
+
+The exterior ring of a GeoJSON polygon _should_ be wound counter-clockwise.
+Taken literally, this means that a clockwise-wound polygon would be a hole in a large polygon that encloses both poles.
+However, in the real word, it's not uncommon for polygons to be wound incorrectly (e.g. [#29](https://github.com/gadomski/antimeridian/issues/29) and [#32](https://github.com/gadomski/antimeridian/issues/32)).
+Our package therefore defaults to correcting the winding order to be counter-clockwise, and only creates these pole-encompassing polygons if explicitly asked:
+
```

Comment:
maybe add something here about most GIS software auto-rewinding these, so it's hard to see that it is rewound, and also that they typically display antimeridian-crossing, correctly-wound polygons as latitude-spanning polygons.

Reply:
I'll add the first bit, but the second bit seems a bit hard to describe well w/o some visual examples. _Technically_ an antimeridian-crossing, ccw-wound Polygon is still incorrect b/c it's not split, so I'm hesitant to get into the weeds describing why the latitude-spanning viz is wrong?

Reply:
ok